src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -1981,6 +1981,33 @@ def _validate_gallery_image_reference(cmd, namespace):
                                     'format, please refer to the help sample'.format(gallery_image_reference))
 
 
+def _validate_gallery_image_reference_by_aaz(namespace):
```

Comment:
May I ask why we need to define a separate function instead of reusing `_validate_gallery_image_reference` function?